chris_backend/plugins/models.py

```diff
@@ -24,6 +24,11 @@
 STATUS_TYPES = ['started', 'running-on-remote', 'finished-on-remote']
 
 class Plugin(models.Model):
+    defaults = {
+        # OpenShift cannot process fractional GPUs as of now.
```

Comment:
Shouldn't have an OpenShift specific comment here.

Reply:
I think, there may be a system that is capable of specific fractional GPUs. As of now, I have given integer type to GPU columns but we may have to change the type to float or smthg else at later point. That was the reason I wrote that comment for my remembrance. I can delete that.